src/transformers/models/layoutlmv2/image_processing_layoutlmv2.py

```diff
@@ -187,7 +187,6 @@ def preprocess(
         tesseract_config: Optional[str] = None,
         return_tensors: Optional[Union[str, TensorType]] = None,
         data_format: ChannelDimension = ChannelDimension.FIRST,
-        **kwargs,
```

Comment:
Doesn't seem related to this PR, why is it here?

Reply:
I have no idea.

Reply:
I think I just bulked reverted some changes I had there at some point. I think I did `git checkout main src/tr... ` maybe my main wasn't up-to-date.